restbridge/detail/RequestHandler.cpp

```diff
@@ -56,13 +55,20 @@ RequestHandler::~RequestHandler()
 {
     _listening = false;
     _thread.join();
-    _requestLock.unlock();
 }
 
 void RequestHandler::listen_()
 {
+    const zeq::Event& vocabularyRequest =
+        zeq::vocabulary::serializeRequest( zeq::vocabulary::EVENT_VOCABULARY );
+
```

Comment:
Why not declare vocabularyRequest variable in the scope of the if statement bellow? It's only used if the subscriber has a vocabulary handler.